samples/session-replay-mobile-schema/session-replay/incremental-snapshot-record.json

```diff
@@ -0,0 +1,53 @@
+{
+    "timestamp": 1657620232337,
+    "type": 3,
+    "data": [
```

Comment:
Here I also have a question, in the browser SDK equivalent, it is not a list but a unique `IncrementalData` and inside the `MutationData` contains the list. Is it a required feature to have list here ? Can we receive a list of mutations at once that are not homogeneous like this example `[MutationData, MousemoveData, MouseInteractionData, ScrollData]` ?

Reply:
Oh I think I see what you say...it might be a problem there...let me look into this. I will come back with a fix.